texttospeech/samples/v1/samplegen_write_bytes_to_file.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "samplegen_write_bytes_to_file")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-texttospeech
+
+# sample-metadata
+#   title: Synthesize an .mp3 file with audio saying the provided phrase
+#   description: Synthesize an .mp3 file with audio saying the provided phrase
+#   usage: python3 samples/v1/samplegen_write_bytes_to_file.py
+
+# [START samplegen_write_bytes_to_file]
+from google.cloud import texttospeech_v1
+from google.cloud.texttospeech_v1 import enums
+
+
+def sample_synthesize_speech():
+    """Synthesize an .mp3 file with audio saying the provided phrase"""
+
+    client = texttospeech_v1.TextToSpeechClient()
+
+    text = "Hello, world!"
+    input_ = {"text": text}
```

Comment:
Should we instead name this "speech_input" or "sample_input" to avoid having to use the underscore notation? that syntax may not be familiar to folks

Reply:
Good question. Not being a Pythonista, myself, I was originally confused by these.

Would it be configurable or... whenever a keyword is used, instead of using a _ suffix, use a prefix instead which scopes the variable to the sample so it's safely not a keyword?

Ack +1

Maybe `request_input`? Trying to think of a prefix that'll work and make sense in as many cases as possible